src/scanpy/preprocessing/_pca/__init__.py

```diff
@@ -471,7 +472,7 @@ def _handle_dask_ml_args(
 def _handle_dask_ml_args(svd_solver: str | None, method: MethodDaskML) -> SvdSolvDaskML:
     import dask_ml.decomposition as dmld
 
-    args: tuple[SvdSolvDaskML, ...]
+    args: AbstractSet[SvdSolvDaskML]
```

Comment:
I would say we should probably use a different name since `AbstractSet` is deprecated (I understand that this is a renaming, but I don't really get why.  I'm fine with the reason for renaming, whatever it is, but just would want something that isn't the same as something that is deprecated).

Reply:
As discussed in private: https://docs.astral.sh/ruff/rules/unaliased-collections-abc-set-import/

Added `PYI` to https://github.com/scverse/anndata/issues/1241